plugin/src/main/java/com/denizenscript/denizen/scripts/commands/entity/EquipCommand.java

```diff
@@ -188,25 +202,24 @@ else if (livingEntity instanceof Steerable) {
             else {
                 LivingEntity livingEntity = entity.getLivingEntity();
                 if (livingEntity != null) {
-                    if (livingEntity instanceof AbstractHorse) {
+                    if (equipment.get("body") != null && (livingEntity instanceof Wolf
+                            || livingEntity instanceof AbstractHorse
+                            || (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_21) && livingEntity instanceof HappyGhast))) {
+                        livingEntity.getEquipment().setItem(EquipmentSlot.BODY, equipment.get("body").getItemStack());

```

Comment:
Can't confirm offhand, but I wonder if these checks are needed? before it was a horse-specific method, but now that it's a general equipment slot can't it technically be set on any entity, like the rest of the equipment slots?